usr.sbin/adduser/adduser.sh

```diff
@@ -197,11 +199,14 @@ save_config() {
 	echo "defaultgroups=$ugroups"	>> ${ADDUSERCONF}
 	echo "passwdtype=$passwdtype" 	>> ${ADDUSERCONF}
 	echo "homeprefix=$homeprefix" 	>> ${ADDUSERCONF}
+	echo "zfs_homeprefix=$zfs_homeprefix" 	>> ${ADDUSERCONF}
```

Comment:
This should always be derived from `homeprefix` instead of passed from command line.

Reply:
In that case, the administrator would be required to pass in a homeprefix that is already mapped to a zfs dataset.  In your opinion, should there be an option where the script is able to create a dataset that's not mapped onto an existing location?